exonum-java-binding-core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -50,43 +50,54 @@ impl JavaServiceRuntime {
         }
     }
 
-    /// Returns internal service proxy.
-    pub fn service_proxy(&self) -> ServiceProxy {
-        self.service_proxy.clone()
-    }
-
-    /// Initializes JVM with provided configuration.
-    ///
-    /// # Panics
-    ///
-    /// - If user specified invalid additional JVM parameters.
-    fn create_java_vm(jvm_config: JvmConfig, ejb_config: EjbConfig) -> JavaVM {
+    /// Builds arguments for JVM initialization.
+    fn build_jvm_arguments(jvm_config: JvmConfig, ejb_config: EjbConfig) -> Result<InitArgs> {
```

Comment:
Please rearrange the methods in the call order, placing this one under its only caller, `create_java_vm`.